src/daemon/daemon.cpp

```diff
@@ -124,7 +124,7 @@ bool t_daemon::run(bool interactive)
       return false;
     mp_internals->rpc.run();
 
-    daemonize::t_command_server* rpc_commands;
+    daemonize::t_command_server* rpc_commands = NULL;
```

Comment:
Can you explain more, anonimal ? NULL is a rvalue and constant. It's typically void* I think, but what is nullptr's type ? I've seen nullptr_t somewhere, is that what you're talking about ?